examples/docs_beta_snippets/docs_beta_snippets/guides/data-assets/data-assets/multi_asset_decorator.py

```diff
@@ -0,0 +1,10 @@
+import dagster as dg
+
+
+@dg.multi_asset(specs=[dg.AssetSpec("asset_one"), dg.AssetSpec("asset_two")])
+def my_multi_asset():
+    # materialize both asset_one and asset_two
```

Comment:
maybe yield a couple of MaterializeResults here

Reply:
added in https://github.com/dagster-io/dagster/pull/24064/commits/ae4866614107b92e5cfd8cc9b3c6bdf1ba786b9a